change/@ni-nimble-components-b897b4de-cdc6-4a5c-84f7-e4a2fc2c9c07.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If option 1 is currently selected, I click the select dropdown arrow, mouse down to option 3, click the select dropdown arrow again. I see the option value does not change (expected) but it looks like a change event is emitted (unexpected)

![dropdownactivation](https://github.com/ni/nimble/assets/1588923/a57132b3-806a-4589-86ea-6688fd1b3446)

Reply:
Fixed.